### PR TITLE
UAF-3711 BUG - PREQ - Receive STE when canceling PREQ documents

### PR DESCRIPTION
**Pull request by mwellman11**

### CODE REVIEW

File: kfs-core/src/main/resources/edu/arizona/kfs/workflow/PaymentRequestDocument.xml [hunk 1]
```diff
@@ -14,6 +14,7 @@
 	      	<status>Initiated</status>
 	      	<status>In Process</status>
 	      	<status>Canceled In Process</status>
+	      	<status>Canceled - In Process</status>
```
Comment (ghost) [inline kfs-core/src/main/resources/edu/arizona/kfs/workflow/PaymentRequestDocument.xml]: Based on the values in DB table DEPR_AP_PMT_RQST_STAT_T and the values in org/kuali/kfs/module/purap/PurapConstants.java, there is only one "canceled in process" valid PURAP document status code for the Payment Request Document. Therefore, "Canceled - In Process" (line 17) should replace "Canceled In Process" (line16).

**Comment by ghost** [comment]
Please see my comment. A little surprised that this fixed the problem. I thought that these lists of document statuses in PURAP document workflow files was only for document search,

File: kfs-core/src/main/resources/edu/arizona/kfs/workflow/PaymentRequestDocument.xml (hunk 1 above)
Comment (mwellman11) [reply kfs-core/src/main/resources/edu/arizona/kfs/workflow/PaymentRequestDocument.xml]: Running an additional test on this change

Comment (mwellman11) [reply kfs-core/src/main/resources/edu/arizona/kfs/workflow/PaymentRequestDocument.xml]: Due to complications caused by the Database Refresh, further work on this is blocked for the time being

Comment (mwellman11) [reply kfs-core/src/main/resources/edu/arizona/kfs/workflow/PaymentRequestDocument.xml]: Line 16 has been removed, process has been tested and works properly

**Comment by ghost** [comment]
Looks good!

**Pull request merged**